tardis/montecarlo/base.py

```diff
@@ -359,16 +359,16 @@ def from_config(cls, config):
             logger.debug("Electron scattering switched on")
             sigma_thomson = 6.652486e-25 / (u.cm ** 2)
 
-        spectrum_frequency = quantity_linspace(
-            config.spectrum.stop.to('Hz', u.spectral()),
-            config.spectrum.start.to('Hz', u.spectral()),
-            num=config.spectrum.num + 1)
-
-        return cls(seed=config.montecarlo.seed,
+        # spectrum_frequency = quantity_linspace(
+        #     config.spectrum.stop.to('Hz', u.spectral()),
+        #     config.spectrum.start.to('Hz', u.spectral()),
+        #     num=config.spectrum.num + 1)
+        spectrum_frequency = config.spectrum.array       
+	return cls(seed=config.montecarlo.seed,
```

Comment:
- Don't comment code that you think is not needed, instead just remove it. 
- config.spectrum.array does not occur from any yaml constructor.
- The purpose was to have a linspace created directly from the `YAMLLoader` through e.g. `config.spectrum` but not only for spectrum.